### PR TITLE
Update prometheus operator permissions to v0.73.2

### PR DESCRIPTION
**Pull request by rene-dekker**

### CODE REVIEW

File: pkg/render/monitor/monitor.go [hunk 1]
```diff
@@ -356,6 +356,23 @@ func (mc *monitorComponent) prometheusOperatorClusterRole() *rbacv1.ClusterRole
...
+			},
+		},
+		{
+			APIGroups: []string{"networking.k8s.io"},
+			Resources: []string{"ingresses"},
+			Verbs: []string{
+				"get",
+				"list",
+				"watch",
+			},
```
Comment (vara2504) [inline pkg/render/monitor/monitor.go:374]: It looks like a duplicate entry, similar to the one in line 351.

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

**Review by vara2504 — APPROVED** [review]
lgtm

**Pull request merged**